### PR TITLE
alt was added to magnifying glass icon.

### PR DESCRIPTION
**Pull request by Ramin-Farhadi**
Alt attribute added to magnifying glass. 

**Types of changes**
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**
- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jmbarne3** [comment]
Can you go ahead and create an rc- branch for this - probably `rc-v0.1.1`. And after this gets merged in, let's hold off on tagging until we're sure there aren't any other accessibility updates we might need to make.

**Review by jmbarne3 — APPROVED** [review]

**Review by kimberspencer — APPROVED** [review]

**Pull request merged**
